src/features/pages/404/content/types.ts

```diff
@@ -0,0 +1,17 @@
+import { TString } from 'types/generics'
+import { LinkUrlType } from 'features/types'
+
+type TInfoItem = {
+    header: TString
+    sub_header: TString
+}
+type TNotAvailableImage = {
+    src: string
+    alt: string
+}
+type TButtonLink = {
+    text: TString
+    url: LinkUrlType
+}
+
+export { TInfoItem, TNotAvailableImage, TButtonLink }
```

Comment:
@NikitK-deriv We shouldn't need this, Let TS infer the types as those variables are root level.

Reply:
We can only remove TNotAvailableImage and due the other have more specific values than just string
Thanks for note @farzin-deriv  👍